bin/ansible-connection

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
-
-# (c) 2017, Ansible, Inc. <support@ansible.com>
```

Comment:
Why remove the copyright line?  If it's because of the email address we could move to the shorter copyright and license notice:
```
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
simple mistake, not intentional